Assets/Scripts/Character/CharacterMovement.cs

```diff
@@ -86,13 +100,13 @@ public class DeathSettings
 	public CharacterController characterController;
 	private Vector3 moveDirection;
     private Statistics statistics;
-	public CharacterAudioController characterAudio;
+	//public CharacterAudioController characterAudio;
```

Comment:
Why is it commented out? Do we still need it?

Reply:
done